tools/xcodeproj_shims/installers/lldb-settings.sh

```diff
@@ -21,10 +21,39 @@ settings set target.sdk-path $SDKROOT
 settings set target.swift-framework-search-paths $FRAMEWORK_SEARCH_PATHS
 END
 
+# Holds all swift-extra-clang-flags
+LLDB_SWIFT_EXTRA_CLANG_FLAGS=()
+# Append extra clang flags if set
 if [[ -n ${BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS:-} ]]
+then
+  LLDB_SWIFT_EXTRA_CLANG_FLAGS+=($BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS)
+fi
+# Append the file 'BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS_FILE' exists
+# it memans that there are framework search paths that LLDB should
+# be aware of so append the contents of that file
+#
+# Look for this logic in 'rules/xcodeproj.bzl'
+if [[ -n "$BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS_FILE" ]]
+then
+  # All framework search paths from file
+  FSPS="$(cat $BAZEL_WORKSPACE_ROOT/$BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS_FILE)"
+  # Xcode won't resolve these paths for us since we're
+  # passing it directly from file to file
+  #
+  # Manually solving 'BAZEL_WORKSPACE_ROOT' and 'PLATFORM_DIR' for now 
+  # until it's possible to pass bigger build settings to Xcode
+  #
+  # For context see: https://github.com/bazel-ios/rules_ios/pull/216
+  FSPS_WITH_ROOT_RESOLVED=${FSPS//"\$BAZEL_WORKSPACE_ROOT"/$BAZEL_WORKSPACE_ROOT}
+  FSPS_WITH_ROOT_RESOLVED=${FSPS_WITH_ROOT_RESOLVED//"\$(PLATFORM_DIR)"/$PLATFORM_DIR}
+  LLDB_SWIFT_EXTRA_CLANG_FLAGS+=($FSPS_WITH_ROOT_RESOLVED)
+fi
```

Comment:
What if instead of this approach, you do the following:

```
for [[ fsp in $FRAMEWORK_SEARCH_PATHS ]]; do
 $LLDB_SWIFT_EXTRA_CLANG_FLAGS += " -F$fsp"
done
```

Would that work? That way you could get rid of the `$BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS_FILE` and `BAZEL_LLDB_FRAMEWORK_SEARCH_PATHS` altogether.

Reply:
You're right! This could work @amberdixon, just one thing concerns me 

We need to assert that all `FRAMEWORK_SEARCH_PATHS` are being passed to LLDB, I'd have to change the check in [post_build_check](https://github.com/bazel-ios/rules_ios/blob/master/tests/ios/xcodeproj/post_build_check.sh) to find the generated `.lldbinit` files and check if it's content contains `FRAMEWORK_SEARCH_PATHS` being set to the correct flag (`swift-extra-clang-flags`)

Atm when Xcode builds and the installer is called we put these files under `CONFIGURATION_TEMP_DIR`, are these generated during the tests here in `rules_ios` so I can find the `.lldbinit` files and figure out a way to inspect `swift-extra-clang-flags`?